__tests__/app.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/healthcheck: HealthCheck to confirm connection with sever", () => {
+  describe("/api/healthcheck", () => {
+    test("200 - returns a status of 200 ", () => {
+      return request(app).get("/api/healthcheck").expect(200);
+    });
+  });
+});
+
+describe("/api/topics", () => {
+  test("status:200, Should get all topics with the corrrect data", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(3);
+        body.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
+
```

Comment:
The following tests rely upon our data coming back in a specific order, which at this point we cannot guarantee. The  test above in this case would've been enough for this ticket